auth.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime, timedelta
+from typing import Optional
+from jose import JWTError, jwt
+from passlib.context import CryptContext
+from sqlalchemy.orm import Session
+from fastapi import Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer
+
+from database import get_db
+from models import User
+
+SECRET_KEY = "a0342e90271446facc8f2b84002a4daa797da99772ad11c3efd488c64706b777"
```

Comment:
Instead of hardcoding the secret_key value, better to use environment variables for the sensitive data.